docs/reference/services.rst

```diff
@@ -4,10 +4,64 @@
 MicroOVN services
 =================
 
+MicroOVN has lots of its functionality separated into distinct services that can
```

Comment:
Drop "has lots of its", `MicroOVN's functionality is ...`

A slight digression no action requested now, our use of the word `service` for multiple concepts is a bit unfortunate, I guess this why sunbeam went with a role concept, but that is for another day.

Let's split the entire page into two sections, leaving the original content intact, apart from changing `page` to `section`.

Like this:
```rst
=================
MicroOVN services
=================

*Your new content*.

snap services
-------------
This section ...
```